resources/kinesisanalytics-streams.go

```diff
@@ -0,0 +1,68 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/kinesisanalytics"
+)
+
+type KinesisAnalyticsStream struct {
```

Comment:
Why do you call it `KinesisAnalyticsStream` instead of `KinesisAnalyticsApplication`?

Reply:
Because I was apparently tired, Ill fix it now